recipe/migrations/cuda110.yaml

```diff
@@ -49,8 +57,8 @@ cxx_compiler_version:   # [linux]
 
 cudnn:
   - undefined
-  - 7                   # [linux64]
-  - 8                   # [linux64]
+  - 7                   # [linux]
+  - 8                   # [linux]
```

Comment:
Does this mean we need to build cuDNN for aarch64/ppc64le first before kicking off the migrator?

Reply:
That's a good question. How have we handled `cudnn` before on new platforms? Would think that `cudnn` would be migrated before any package that depends on it, but it is always possible things get jammed up before it gets there